crates/kernel/testsuite/moot/eval/nonprogrammer.moot

```diff
@@ -1,4 +1,4 @@
-# non-programmers shouldn't be able to execute the eval trampoline
+// non-programmers shouldn't be able to execute the eval trampoline
```

Comment:
"shouldn't be able to set programmer bit"

Reply:
Changed to use `@nonprogrammer` in a new commit